component/layout/AdminLayout.jsx

```diff
@@ -47,17 +47,17 @@ function AdminLayout(props) {
       .then((response) => {
         if (response.status === 200) {
           setUserOrganizationData({
-            orgId: response?.data?.organization?.company_name,
+            orgId: response.data?.organization?.company_name,
             roleId:
-              response?.data?.roleData?.role_id === 2
+              response.data?.roleData?.role_id === 2
                 ? "Admin"
-                : response?.data?.roleData?.role_id === 3
+                : response.data?.roleData?.role_id === 3
                 ? "Manager"
-                : response?.data?.roleData?.role_id === 4
+                : response.data?.roleData?.role_id === 4
                 ? "Member"
                 : null,
-            role: response?.data?.roleData?.role_id,
-            userImage: response?.data?.UserDetails?.image,
+            role: response.data?.roleData?.role_id,
+            userImage: response.data?.UserDetails?.image,
```

Comment:
Remove Null checks for data because is not data, User need to logout as if no organisation is returned